src/app/components/pool/action/add/pool-create-basic-dialog.component.ts

```diff
@@ -38,6 +39,30 @@ export class PoolCreateBasicDialogComponent extends DynamicForm<Pool, PoolCreate
         return this._osControl.value && this._osControl.value.cloudServiceConfiguration;
     }
 
+    public get isSelectedImageDeprecated() {
+        if (this._osControl.value && this._osControl.value.virtualMachineConfiguration) {
```

Comment:
FYI, Typescript has this great syntax now, where it will return undefined if `value` is undefined instead of erroring out:

```
const config = this._osControl.value?.virtualMachineConfiguration;
if (config) {
   // ...etc
}
```

Reply:
(No need to change this unless you want)

Reply:
Updated to the suggested syntax